scheduler/src/cook/scheduler/constraints.clj

```diff
@@ -90,6 +90,14 @@
   (let [previous-hosts (job->previous-hosts-to-avoid job)]
     (->novel-host-constraint job previous-hosts)))
 
+(defn job->gpu-model-requested
+  "Get GPU model requested from job environment or use default GPU model on pool"
+  [gpu-count gpu-model-in-env pool-name]
```

Comment:
Instead of passing in `gpu-model-in-env`, it seems like you could just pass in the job and have this function do the `(-> job util/job-ent->env (get "COOK_GPU_MODEL"))` part. That would let you get rid of the `api/get-gpu-model-from-task-metadata` function as well.

Reply:
good call, implemented.